schema.csv

```diff
@@ -64,11 +64,11 @@ file.size,long,extended,
 file.target_path,keyword,extended,
 file.type,keyword,extended,
 file.uid,keyword,extended,
-geo.city_name,keyword,core,
-geo.continent_name,keyword,core,
-geo.country_iso_code,keyword,core,
-geo.location,geo_point,core,
-geo.region_name,keyword,core,
+geo.city_name,keyword,core,Montreal
```

Comment:
Thanks for adding the example. Perhaps in the future make separate PR's for these changes as then we can get them in quickly and it becomes more obvious which parts in the auto generated files are affected by this PR.

Reply:
I agree in principle. Although there are so many changes to do still, that I'm trying to lump some related things together, to speed up the overall process (even though the examples in this case will be delayed slightly).